hoomd/metal/EAMForceCompute.cc

```diff
@@ -355,9 +355,6 @@ void EAMForceCompute::computeForces(uint64_t timestep)
     // create a temporary copy of r_cut squared
     Scalar r_cut_sq = m_r_cut * m_r_cut;
 
-    // sum up the number of forces calculated
-    int64_t n_calc = 0;
```

Comment:
Why the removals here in metal?

Reply:
The latest Apple clang generates warnings for the unused `n_calc`. This change is unrelated and could be cherry-picked into another pull request if necessary.

Reply:
I also found a few more unused `benchmark` functions and removed them. The removal can be cherry-picked to another branch if needed.

Reply:
No need to worry about that then. Just wondering what prompted the seemingly unrelated change.

Reply:
👍 I wondered about benchmark removal too